python/res/fm/shell/__init__.py

```diff
@@ -0,0 +1 @@
+from .shell import Shell
```

Comment:
Good if you in this `__init__` file also add small functions like:
```
def move_file(src,target):
      Shell.moveFile(src,target)
```

so that everything can be called as:
```python
from res.fm.shell import move_file
...
move_file(f1,f2)
```